addons/payment/migrations/13.0.1.0/noupdate_changes.xml

```diff
@@ -19,37 +19,37 @@
         <li class="list-inline-item"><i class="fa fa-check"/>Embedded Credit Card Form</li>
       </ul>
     </field>
-    <field name="image" type="base64" file="payment_authorize/static/src/img/authorize_icon.png"/>
+<!--    <field name="image" type="base64" file="payment_authorize/static/src/img/authorize_icon.png"/>-->
     <field name="display_as">Credit Card (powered by Authorize)</field>
     <field name="image_128" type="base64" file="payment_authorize/static/src/img/authorize_icon.png"/>
   </record>
   <record id="payment_acquirer_buckaroo" model="payment.acquirer">
-    <field name="image" type="base64" file="payment_buckaroo/static/src/img/buckaroo_icon.png"/>
+<!--    <field name="image" type="base64" file="payment_buckaroo/static/src/img/buckaroo_icon.png"/>-->
     <field name="display_as">Credit Card (powered by Buckaroo)</field>
     <field name="image_128" type="base64" file="payment_buckaroo/static/src/img/buckaroo_icon.png"/>
   </record>
   <record id="payment_acquirer_paypal" model="payment.acquirer">
```

Comment:
you have to manually add the noupdate changes for the payment_acquirer_ingenico. An easy way to do that, is to manually rename the payment_acquirer_ogone to payment_acquirer_ingenico in a local v12 and then run the noupdate generator script in v13

Reply:
thanks. done